common/utils.go

```diff
@@ -22,8 +24,55 @@ func NewClient(ctx context.Context, config NewClientConfig) (*spanner.Client, er
 	}
 	if config.Endpoint != "" {
 		options = append(options, option.WithEndpoint(config.Endpoint))
-		sdk.Logger(ctx).Info().Msg("using custom endpoint")
 	}
 
 	return spanner.NewClient(ctx, config.DatabaseName, options...)
 }
+
+func NewDatabaseAdminClient(ctx context.Context) (*database.DatabaseAdminClient, error) {
```

Comment:
No need to export.

Reply:
Hum, it is exported because I did plan to add the combined iterator, which would use this admin client. I guess we can just delete this function until I add it.